src/devtoolsPanel.ts

```diff
@@ -180,6 +182,16 @@ export class DevToolsPanel {
         this.timeStart = performance.now();
     }
 
+    private onToggleCSSMirrorContent(message: string) {
+        const { isEnabled } = JSON.parse(message) as IToggleCSSMirrorContentData;
+
+        if (isEnabled) {
+            void vscode.commands.executeCommand(`${SETTINGS_VIEW_NAME}.cssMirrorContentOn`);
```

Comment:
We can probably just create one command to toggle it on/off.  We can pass in parameters like this:
```
void vscode.commands.executeCommand(`${SETTINGS_VIEW_NAME}.toggleInspect`, { enabled: false });
```

Reply:
Please take another look.